app/views/api/v3/analytics/user_analytics/_monthly_breakdown.html.erb

```diff
@@ -1,21 +1,16 @@
-<% if current_facility.diabetes_enabled? %>
-  <%= render(Reports::MonthlyProgressComponent.new(facility: current_facility, 
-    diagnosis: :all, 
-    metric: metric,
-    query: @query,
-    range: @range,
-    gender_groups: false)) %>
-
-  <%= render(Reports::MonthlyProgressComponent.new(facility: current_facility, 
-    diagnosis: :diabetes, 
-    metric: metric,
-    query: @query,
-    range: @range)) %>
 
+<% 
+# handle the special case of any diagnosis and all genders first
```

Comment:
I'm realizing this should get moved into the controller, or a new service object.

Reply:
Going to do in a follow up PR.